packages/content/src/Infrastructure/Sulu/Route/ContentRouteDefaultsProvider.php

```diff
@@ -58,16 +59,23 @@ class ContentRouteDefaultsProvider implements RouteDefaultsProviderInterface
      */
     private $cacheLifetimeResolver;
 
+    /**
+     * @var CacheLifetimeRequestStore
+     */
+    private $cacheLifetimeRequestStore;
```

Comment:
maybe we not even need this as the `cacheLifetimeRequestStore` already keeps the `_cacheLifeTime` request attribute in mind. Need to check this next week.